bioptim/limits/path_conditions.py

```diff
@@ -401,6 +401,19 @@ def concatenate(self, other: "Bounds"):
         self.extra_params = self.min.extra_params
         self.n_shooting = self.min.n_shooting
 
+    def scale(self, scaling: Union[float, np.ndarray]) -> "Bounds":
+        """
+        Scaling a Bound
+
+        Parameters
+        ----------
+        scaling: float
+            The scaling factor
+        """
+        self.min = self.min / scaling
+        self.max = self.max / scaling
+        return self
```

Comment:
This should not return anything

Reply:
idem :
https://github.com/fbailly/bioptim/blob/9d75595998618004794e2f1b83cd0315fdaca2a2/bioptim/optimization/parameters.py#L79